kythe/typescript/testdata/class.ts

```diff
@@ -27,10 +27,10 @@ class Class implements IFace {
     //- !{ Member.tag/static _ }
     member: number;
 
-    //- @staticMember defines/binding StaticMember
+    //- @member defines/binding StaticMember
     //- StaticMember.tag/static _
     //- StaticMember childof Class
-    static staticMember: number;
+    static member: number;
```

Comment:
There's a way in the test language to assert that StaticMember != Member, which is important here.